.github/workflows/opensearch-observability-test-and-build-workflow.yml

```diff
@@ -3,8 +3,8 @@ name: Test and Build OpenSearch Observability Backend Plugin
 on: [pull_request, push]
 
 env:
-  OPENSEARCH_VERSION: '1.2.3-SNAPSHOT'
-  OPENSEARCH_BRANCH: '1.2'
+  OPENSEARCH_VERSION: '1.3.0-SNAPSHOT'
+  OPENSEARCH_BRANCH: '1.3'
   COMMON_UTILS_BRANCH: 'main'
```

Comment:
you can remove `OPENSEARCH_BRANCH` and `COMMON_UTILS_BRANCH`, we don't need them in CI anymore since they are available through mavenCentral

Reply:
I'll make sure to remove them in the next PR, don't want to commit anything extra and dismiss your approval on this PR